client-v2/src/components/FrontsEdit/Fronts.js

```diff
@@ -50,6 +58,9 @@ class Fronts extends React.Component<FrontsComponentProps, ComponentState> {
         nextProps.selectedFront.collections
       );
     }
+    if (this.props.frontId !== nextProps.frontId || !this.props.lastPressed) {
+      this.props.frontsActions.fetchLastPressed(nextProps.frontId);
```

Comment:
Should we be polling this endpoint?

Reply:
... yes. Yes we should 😁

Reply:
Let's do this in a separate pr as discussed.